### ISSUE TITLE
"Download as pdf..." missing glvis figures

### ISSUE THREAD

**Issue by rw-anderson** [GLVis/pyglvis#22]
I find these notebooks to be super useful for exploring algorithm alternatives. But since not everyone uses them or is even set up to use them, an export to pdf capability would be really useful for sharing results with others. The notebooks do export to pdf, but the glvis windows just show up with the text "glvis()" in place of the figure.

Would it be feasible to export the figure in the pdf?

**Comment by tzanio** [comment]
I agree that this will be awesome to support... @tomstitt and @kanye-quest, is it possibly related to #5?

**Comment by tomstitt** [comment]
I think they are definitely related (being able to serialize the widget to a png or other format) but I thought someone like matplotlib would do this but they don't if you use the widget version:

![Screen Shot 2021-08-19 at 8 29 55 AM](https://user-images.githubusercontent.com/7017223/130097720-dbf7bcf8-29b5-4923-bfe2-8895de03392c.png)

Regardless this is a good time to revisit screen shots so I'll look into that.

Also, [here is a good discussion on the ideas](https://github.com/LGSInnovations/jupyter-sigplot/issues/26), I think just aiming for a static image would be powerful

**Comment by tzanio** [comment]
Wait... imagine glvis.js embedded in the file and you being able to interact with it in the PDF

🤯

**Comment by tzanio** [comment]
Maybe I am missing something, but printing from Binder, e.g. https://mybinder.org/v2/gh/GLVis/pyglvis/HEAD?filepath=examples%2Fbasic.ipynb, worked for me:

[basic - Jupyter Notebook.pdf](https://github.com/GLVis/pyglvis/files/7042142/basic.-.Jupyter.Notebook.pdf)

**Comment by rw-anderson** [comment]
No, I think it was I who was missing something - using the browser! I was trying to use the notebook export. But the browser print as pdf does exactly what I want.

**Comment by tzanio** [comment]
Awesome! Can we close this then?
